extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -153,6 +153,7 @@ public class VertxHttpRecorder {
         @Override
         public void handle(HttpServerRequest httpServerRequest) {
             if (!uriValid(httpServerRequest)) {
+                LOGGER.error("Server request had bad URI.");
```

Comment:
As already mentioned, we need this one gone. We cannot log something when a URL is badly crafted, this is a security hazard.